modules/ts/src/ts_perf.cpp

```diff
@@ -1691,6 +1943,10 @@ void TestBase::RunPerfTestBody()
 {
     try
     {
+#ifdef ENABLE_INSTRUMENTATION
+        cv::instr::setFlags(cv::instr::getFlags()|cv::instr::FLAGS_MAPPING); // enable mapping for first run
+        cv::instr::queueCommand(cv::instr::COMMAND_RESET); // delayed clean up. actual execution before timer start
```

Comment:
This is to to clear trace from non-test functions before actual testing loop begin. I made it delayed to avoid duplication of test run macros for instrumented case.